### PR TITLE
 pythonPackages.manhole: init at 1.5.0

### PR DESCRIPTION
**Pull request by ivan**
###### Motivation for this change

[manhole](https://github.com/ionelmc/python-manhole) is generally useful for connecting to and debugging long-running Python programs.

This dependency will be used by an upcoming PR that adds [grab-site](https://github.com/ludios/grab-site).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

I tested `pythonPackages.manhole` and `python37Packages.manhole` on NixOS.

### CODE REVIEW

File: pkgs/development/python-modules/manhole/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+buildPythonPackage rec {
+  pname = "manhole";
+  version = "1.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "11ivy8qiv87jl2lc1ldhv9dc4jwf3hz7wysdfiagdcd9kkd48v8m";
+  };
+
+  checkInputs = [ pytest requests process-tests ];
```
Comment (dotlambda) [inline pkgs/development/python-modules/manhole/default.nix]: I suppose `requests` is not needed anymore since you disabled the test ?

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.manhole python3.pkgs.manhole

**Comment by dotlambda** [comment]
Seems like there is a problem on Darwin: https://github.com/NixOS/nixpkgs/pull/51929/checks?check_run_id=40720694

**Comment by ivan** [comment]
Thanks for the review!

I can't reproduce the `/tmp/manhole-socket` removal failure on my macOS 10.13 machine, but it looks like I need to disable the tests anyway: another test fails, suggesting upstream doesn't run them on macOS.

Updated with tests disabled on Darwin.

**Comment by ivan** [comment]
I changed `!stdenv.isDarwin` to `stdenv.isLinux`. Should be done now.

**Comment by rasendubi** [comment]
@GrahamcOfBorg build python27Packages.manhole python37Packages.manhole

**Pull request merged**